webpack.config.js

```diff
@@ -64,6 +65,35 @@ const stylesTransform = ( content ) => {
 	return content;
 };
 
+const blockNameRegex = new RegExp( /(?<=src\/).*(?=(\/frontend))/g );
+
+const createEntrypoints = () => {
+	const scriptPaths = fastGlob.sync(
+		'./packages/block-library/src/**/frontend.js',
+		{
+			ignore: [ '**/build*/**' ],
```

Comment:
Why was it necessary? There are no build folders inside the `block-library/src/` folder or do I miss something?

Reply:
It was necessary at some point, when instead of going into `/src`, I would start from `block-library/`, removing in https://github.com/WordPress/gutenberg/pull/30629